services/delegation/src/modules/blockchain/blockchain.service.ts

```diff
@@ -24,6 +24,9 @@ export class BlockchainService {
       '0x00000000000000000000000000000000000000fe',
       [
         'function delegate(address validator) payable',
+        'function undelegate(address validator, uint256 amount) external',
+        'function confirmUndelegate(address validator) external',
+        'function cancelUndelegate(address validator) external',
```

Comment:
@rattrap  I saw that there is this func too, shouldn't we use this and create another cron to clean up the "outdated" delegations (like the ones older than 5 days)?

Reply:
No. Only we do undelegations. And if we call `undelegate`, we want to confirm it, not cancel it.

Reply:
Yeah, that's right, but won't there be any "dangling" ones after some time?